docs/chat-design.md

```diff
@@ -11,14 +11,14 @@ Ultimately, we aim to innovate without compromising on simplicity and ensure tha
 
 Basic layout consists of the following components:
 
-* Header: component where customisable logo element is displayed. This component also contains: Navigation bar button, Prompts bar button and user profile.
-* Navigation bar: contains all active chats and a button to start a new one.
-* Prompts bar: contains user prompts and an action to add a new one.
-* Conversation area: are where user questions and LLM responds will take place.
+* Header: component where customizable logo element is displayed. This component also contains: Conversations section button, Prompts section button and a user profile section.
```

Comment:
In DIAL these panels are named (the names appear on hover):
Conversation list and Prompt list
Sometimes in manual test cases I name them as 'Conversation list' panel or 'Prompt list' panel.
It sound like not important thing, so JFYI :)

Reply:
fixed